tests/integration/aof.tcl

```diff
@@ -485,4 +485,30 @@ tags {"aof external:skip"} {
         catch {exec src/redis-check-aof --truncate-to-timestamp 1628217469 $aof_path} e
         assert_match {*aborting*} $e
     }
+
+    test {EVAL timeout with slow verbatim Lua script from AOF} {
+        create_aof {
+            append_to_aof [formatCommand select 9]
+            append_to_aof [formatCommand eval {redis.call('set',KEYS[1],'y'); for i=1,1500000 do redis.call('ping') end return 'ok'} 1 x]
```

Comment:
Is there any better way create a delay in the script than a 1.5 million iteration loop? If we want to check a script timeout then perhaps create an infinite loop?

Reply:
infinite loop: Maybe no, for we expect the script **finishes** but its running time greater than `lua-time-limit`? This case is modified from a test here https://github.com/redis/redis/pull/9812#discussion_r762519528 mentioned